### PR TITLE
{bio}[foss/2016b] LocARNA 1.9.2

### PR DESCRIPTION
**Pull request by SethosII**
This adds LocARNA.

### CODE REVIEW

File: easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ analyzing sets of related RNAs without known common structure."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/s-will/%(name)s/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['14aabc5425f5090bf1bcaa61c2db41f659e55dd40100db117fb9e40470b4a2b5']
+
+configopts = '--with-vrna=$EBROOTVIENNARNA'
```
Comment (boegel) [inline easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2-foss-2016b.eb]: style nitpicking: please move this down, below `dependencies`

Comment (SethosII) [reply easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2-foss-2016b.eb]: @boegel I changed this.

File: easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2-foss-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ analyzing sets of related RNAs without known common structure."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/s-will/%(name)s/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['14aabc5425f5090bf1bcaa61c2db41f659e55dd40100db117fb9e40470b4a2b5']
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2-foss-2016b.eb]: @SethosII Please drop the duplicate empty line...

Comment (SethosII) [reply easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2-foss-2016b.eb]: @boegel Woops, changed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c5949ce2ebc1f97ea1a7c87d41096e11 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2004.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/776648b5648f07a57d52201d6b1d906f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**